marley_cake_adventure.py

```diff
@@ -0,0 +1,76 @@
+"""
+author: kalez43@github.com
+description: This is a text-based game. The goal is to make the best decisions in order to allow Marley (dog) to
+retrieve a cake off of the counter without Kayleigh (her human) catching her.
+"""
+
+
+# Space between story strings
+def space():
+    print("")
+
+
+def stars():
+    space()
+    print("* * * * *")
+    space()
+
+
+# Enter function between story telling strings
+def enter():
+    space()

```

Comment:
Won't this while loop end after the first iteration?

Reply:
Yes, I wanted a way for the user to have to press enter in order to move to the next sequence